mise.toml

```
@@ -1,5 +1,5 @@
 [tools]
-node = "latest"
-python = "latest"
-go = "latest"
-ruby = "latest"
+node = "25"
+python = "3.14"
+go = "1.25"
+ruby = "3.4"
```

I like the idea of keeping these to `latest` (unless there's some reason not to). This way, we won't have to remember to bump these versions when new things come out (which I forget all the time), local dev's a bit less confusing, and it's a nice default for us in CI to ensure that we're always supporting whatever's most current.

```suggestion
node = "latest"
python = "latest"
go = "latest"
ruby = "latest"
```